MediaManager/Plugin.MediaManager.Android/MediaPlayerService/MediaPlayerService.cs

```diff
@@ -101,7 +101,11 @@ public override StartCommandResult OnStartCommand(Intent intent, StartCommandFla
         public override async Task Play(IMediaFile mediaFile = null)
         {
             await base.Play(mediaFile);
-            _mediaPlayer.PrepareAsync();
+            try
+            {
+                _mediaPlayer.PrepareAsync();
+            }
+            catch (Exception) { }
```

Comment:
Shouldn't we catch the `IllegalStateException` and try to get the library into a state where we can do something? Maybe it's on a state where you'd have to do something - maybe it's on `STOP` ...

What about all kind of other exceptions that're swallowed here?